data_validation/result_handlers/bigquery.py

```diff
@@ -48,20 +55,25 @@ def get_handler_for_project(
             credentials (google.auth.credentials.Credentials):
                 Explicit credentials to use in case default credentials
                 aren't working properly.
+            status_list (list): provided status to filter the results with
         """
         info = client_info.get_http_client_info()
         client = bigquery.Client(
             project=project_id, client_info=info, credentials=credentials
         )
-        return BigQueryResultHandler(client, table_id=table_id)
+        return BigQueryResultHandler(client, status_list, table_id=table_id)
```

Comment:
nit: to keep the same format, we can pass in the keyword arg for status_list or remove keyword args from table_id:

return BigQueryResultHandler(client, status_list=status_list, table_id=table_id)